kitsune/tidings/tasks.py

```diff
@@ -1,13 +1,55 @@
+from collections.abc import Sequence
+
 from celery import shared_task
+from django.contrib.auth import get_user_model
 
 from kitsune.tidings.models import Watch
+from kitsune.tidings.utils import get_class
 
 
 @shared_task
-def claim_watches(user):
-    """Attach any anonymous watches having a user's email to that user.
-
+def claim_watches(user_id):
+    """
+    Attach any anonymous watches having a user's email to that user.
     Call this from your user registration process if you like.
-
     """
+    user = get_user_model().objects.get(id=user_id)
     Watch.objects.filter(email=user.email).update(email=None, user=user)
+
+
+@shared_task
+def fire(
+    event_cls_module_name,
+    event_cls_name,
+    instance_cls_module_name=None,
+    instance_cls_name=None,
+    instance_id=None,
+    exclude_user_ids=None,
```

Comment:
The calling signature and the actual function signature are not the same. There's an implicit unpacking happening here but the `kwargs` pass to this function from the `Event` class could be anything. Passing `kwargs` is probably cleaner and you can do the unpacking within the function